tools/kdc/kdc.json

```diff
@@ -7,7 +7,7 @@
   "container": {
     "type": "MESOS",
     "docker": {
-      "image": "nvaziri/kdc:mesosphere",
+      "image": "icharalampidis/projects:kdc",
```

Comment:
Noooooooo :-)

Reply:
Yeah, don't worry about this, when I have the review I will set-up a jenkins job and a proper repository 😄 use this if you want to play around.

Reply:
I bet Nima said the same thing :-P

Reply:
So, I am now using `mesosphere/kdc:<commit>` and also created a TS job for this : https://teamcity.mesosphere.io/viewType.html?buildTypeId=DcosIo_DcosCommons_Tools_DcosCommonsKdcImage_Publish&branch_DcosIo_DcosCommons_Tools_DcosCommonsKdcImage=ic%2Fkdc-admin-api&&&tab=buildTypeStatusDiv